### PR TITLE
Append app name to title

### LINKED ISSUE #10899: Feature: Append app name to title
### What feature or improvement do you think would benefit Files?

Unlike UWP, setting the window title in WinUI 3 doesn't append the app name. This makes it harder to distinguish a folder window opened in Files from one opened in File Explorer.

### Requirements

When setting title, set it as folder name - Files.

### Files Version

2.4.0.0

### Windows Version

10.0.22000.1335

### Comments

_No response_

### PR DESCRIPTION
**Pull request by JaiganeshKumaran**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #10899

Instead of hardcoding the app name "Files", it takes the name from the SystemInformation class instead. In dev mode the title will be "Folder name - Files - Dev". I would recommend changing "Files - Dev" to "Files (Dev)" or simply "Files Dev" because it's cleaner.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/ViewModels/MainPageViewModel.cs [hunk 1]
```diff
@@ -233,7 +234,7 @@ public async void UpdateInstanceProperties(object navigationArg)
 			if (AppInstances.Count > 1)
 				windowTitle = $"{windowTitle} ({AppInstances.Count})";
 			if (navigationArg == SelectedTabItem?.TabItemArguments?.NavigationArg)
-				App.GetAppWindow(App.Window).Title = windowTitle;
+				App.GetAppWindow(App.Window).Title = $"{windowTitle} - {SystemInformation.Instance.ApplicationName}";
```
Comment (yaira2) [inline src/Files.App/ViewModels/MainPageViewModel.cs:237]:
```suggestion
				App.GetAppWindow(App.Window).Title = $"{windowTitle} - Files";
```

File: src/Files.App/MainWindow.xaml.cs [hunk 2]
```diff
@@ -45,7 +46,7 @@ public MainWindow()
 		private void EnsureEarlyWindow()
 		{
 			// Set title
-			AppWindow.Title = "Files";
+			AppWindow.Title = SystemInformation.Instance.ApplicationName;
```
Comment (yaira2) [inline src/Files.App/MainWindow.xaml.cs:49]:
```suggestion
			AppWindow.Title = "Files";
```

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/MainWindow.xaml.cs (hunk 2 above)
Comment (JaiganeshKumaran) [reply src/Files.App/MainWindow.xaml.cs]: Why? I do want it to distinguish between Dev, Preview and the release version.

Comment (yaira2) [reply src/Files.App/MainWindow.xaml.cs]: The icon helps with that

**Pull request merged**